src/test_board.py

```diff
@@ -0,0 +1,218 @@
+import unittest
+from board import Board
+
+
+class TestBoard(unittest.TestCase):
+
+    def test_empty_board_should_have_width_x_height_nodes(self):
```

Comment:
Whouah clean test files ! Function name is clean :+1:

Reply:
thanks ;-)